packages/editor/src/client/panel/articleMetadataPanel.tsx

```diff
@@ -380,6 +421,16 @@ export function ArticleMetadataPanel({
                     setAuthorsFilter('')
                   }}
                   block
+                  renderExtraFooter={() =>
+                    authorsFilter &&
+                    data?.authors.nodes.length === 0 && (
+                      <div style={{float: 'right', margin: '20px'}}>
```

Comment:
I guess using `float` has side effects and you will need to clearfix the parent. you can use flex for example what do you think?

Reply:
It's not needed anymore since I changed the button's name. 👍